client/post-editor/editor-post-formats/accordion.jsx

```diff
@@ -1,68 +1,64 @@
 /**
  * External dependencies
  */
-var React = require( 'react' ),
-	pick = require( 'lodash/pick' ),
-	find = require( 'lodash/find' ),
-	classNames = require( 'classnames' ),
-	isEmpty = require( 'lodash/isEmpty' );
+import React, { PropTypes } from 'react';
+import { has, isEmpty } from 'lodash';
+import classNames from 'classnames';
 
 /**
  * Internal dependencies
  */
-var Accordion = require( 'components/accordion' ),
-	Gridicon = require( 'components/gridicon' ),
-	siteUtils = require( 'lib/site/utils' ),
-	PostFormats = require( './' );
+import Accordion from 'components/accordion';
+import Gridicon from 'components/gridicon';
+import siteUtils from 'lib/site/utils';
+import PostFormats from './';
 
-module.exports = React.createClass( {
+export default React.createClass( {
 	displayName: 'EditorPostFormatsAccordion',
 
 	propTypes: {
-		site: React.PropTypes.object,
-		post: React.PropTypes.object,
-		postFormats: React.PropTypes.arrayOf( React.PropTypes.shape( {
-			slug: React.PropTypes.string,
-			label: React.PropTypes.string
-		} ) )
```

Comment:
I suppose the change of the `postFormats` type is due to the return value of the `getPostFormats` selector. I was wondering whether we should add something like `Object.values(postFormats)` inside the selector to get an array instead of an object.

I know the object can have better performance when accessing postFormats by slug, but it feels more "natural"  to have an array of postFormats when we call a function called `getPostFormats`, don't you think ?